wagtail/project_template/media/models.py

```diff
@@ -0,0 +1,58 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from wagtail.documents.models import AbstractDocument
+from wagtail.images.models import AbstractImage, AbstractRendition
+
+from taggit.managers import TaggableManager
+
+
+class Image(AbstractImage):
+    admin_form_fields = (
+        'title',
+        'file',
+        'collection',
+        'tags',
+        'default_alt_text'
```

Comment:
should this be 'generic_alt_text'?

Reply:
I’d prefer we leave this out entirely. I’d love to improve how we do alt text but would rather approach it more strategically ([RFC 51](https://github.com/wagtail/rfcs/pull/51) or similar), based on our best understanding possible of the best practices.

In my opinion, this PR would be plenty valuable enough even if the custom models had no custom fields defined in the base template.